tests/helpers.py

```diff
@@ -294,6 +302,24 @@ def create_pvc(sc_name, pvc_name=None, size=None, wait=True):
     return ocs_obj
 
 
+def create_multiple_pvc(sc_name, namespace, number_of_pvc=1, size=None, ):
+    """
+    Create one or more PVC
+
+    Args:
+        sc_name (str): The name of the storage class to provision the PVCs from
+        number_of_pvc (int): Number of PVCs to be created
```

Comment:
please order the args like passed to the function

Reply:
done